spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -323,7 +330,9 @@ def _start_download(self):
         self.set_status(DOWNLOADING_INSTALLER)
 
         self.progress_dialog = ProgressDialog(
-            self, _("Downloading Spyder {} ...").format(self.latest_release)
+            self,
+            _("Downloading update for Spyder {} ...").format(
+                self.latest_release)
```

Comment:
Shown for both the installer download and conda-lock file download.
If this is not desirable, then I'll rework to bypass the progress dialog for the conda-lock file.

Reply:
I think it's ok to show it for the installer but not when downloading the conda-lock file because I guess that takes a few seconds, right?

Please check that manually and if you see the UX is that dialog is shown and hidden almost immediately when doing updates with conda-lock, then only leave it for the installer.

Reply:
Also, I reverted the progress dialog text.